<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QMainWindow" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="background">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="input_city">
            <property name="styleSheet">
             <string notr="true">background-color:#111;
border:1px solid #111;
color:white;
border-radius:5px;
padding:2px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 170, 0);
color:white;
border:1px solid rgb(0, 170, 0);
border-radius:5px;
padding:2px;</string>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="showWeather_today">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QWidget" name="showWeather_todayPage1" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QFrame" name="frame">
             <property name="styleSheet">
              <string notr="true">QFrame{
background-color:rgb(0, 170, 255);
color:white;
border:1px solid #111;
border-radius:10px;}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="show_icon">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border:none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="show_temp">
                <property name="font">
                 <font>
                  <pointsize>48</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border :none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="show_loc">
                <property name="styleSheet">
                 <string notr="true">border:none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="max_temp">
                <property name="styleSheet">
                 <string notr="true">border:none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="min_temp">
                <property name="styleSheet">
                 <string notr="true">border:none;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
